python_modules/dagster/dagster/_core/execution/context/system.py

```diff
@@ -386,68 +375,6 @@ def has_partitions(self) -> bool:
             )
         )
 
-    @property
-    def partition_key(self) -> str:
```

Comment:
No changes to this method, just moved from `PlanExecutionContext` to `StepExecutionContext` so it can look at the current node.